app/models/course/assessment/submission.rb

```diff
@@ -167,6 +167,11 @@ def latest_answers
     answers.group_by(&:question_id).map { |pair| pair[1].last }
   end
 
+  # The answers with current_answer flag set to true.
+  def current_answers
```

Comment:
Do we still need the `latest_answers` ?

Reply:
They're needed in `check_zombie_job` in the submissions controller.

Reply:
There's one `latest_answers` scope in the answers concern also, how about that?